package.json

```diff
@@ -24,7 +24,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "1.56.0-insider"
+        "vscode": "1.56.0"
```

Comment:
Can't fully remember our process on this engine number. Don't we keep it as -insider in main?

Reply:
Yeah should be 1.57.0-insider

Reply:
I think we discussed this at bug bash and said we don't need to bump this to 1.57.0-insider right away, as that will block publishing nightly insiders builds until 1.57.0-insider becomes available

Reply:
But yes good point, closing this PR since I can't fix this directly in the release branch (which is the source of this PR)